pkgs/development/interpreters/python/default.nix

```diff
@@ -99,7 +99,8 @@ with pkgs;
         inherit sourceVersion;
         pythonAtLeast = lib.versionAtLeast pythonVersion;
         pythonOlder = lib.versionOlder pythonVersion;
-        inherit hasDistutilsCxxPatch pythonForBuild;
+        inherit hasDistutilsCxxPatch;
+        pythonForBuild = pythonPackagesBuildHost;
```

Comment:
I don't think this is right?

Reply:
Why not? `passthruFun` is called with `pythonPackagesBuildHost = pkgsBuildHost.${pythonAttr};`.

Reply:
we had `pythonPackagesBuildHost == pythonForBuild.pkgs` before, i.e. `pythonForBuild` was a derivation

Reply:
`pythonForBuild` is a derivation, and `pkgs` is the Python package set it comes with.